build/docker/Dockerfile.grafana

```diff
@@ -25,4 +25,7 @@ ARG PLUGINS=""
 # Install each plugin.
```

Comment:
add code in there:
```dockerfile
IFS=','
```

[The Meaning of IFS in Bash](https://www.baeldung.com/linux/ifs-shell-variable)